docs/SchemaMigrationGuide.md

```diff
@@ -1,53 +1,54 @@
 # Schema Migration Guide
-This document describes how to perform Sql schema migration on the application server.
+As Microsoft continues to support the newer releases of FHIR, migrations for the FHIR Server for Azure SQL Server edition will be required.
 
- A database schema will evolve over time. The corresponding server in production would need to be upgraded to the next/latest version of the code, which may require schema migration.
+This document describes how to perform SQL Server schema migration on the FHIR Server for Azure application server.
 
-A migration is a T-SQL script that alters the database in some way. The script has a version encoded in its file name. This version is an incrementing integer.
+ The database schemas will evolve with each release of new features and editions of FHIR. The corresponding FHIR Server for Azure would need to be upgraded to the next/latest version of the schema, which may require migration of the SQL Server schema.
 
-## There are two ways to upgrade the schema
+The migration is a T-SQL script which alters the database. The script filename is encoded with the version. For example - The version is an incrementing integer.
+
+## Methods for upgrading
 1. Automatic schema upgrade
 
-2. Schema upgrade via schema migration tool
+2. Schema upgrade via SQL Server schema migration tool
 
-## 1. Automatic Schema Upgrade
-Here, the schema migration is performed automatically on the server startup.
-The scripts that upgrade schema are part of any upgrade package for the application and run automatically when the application launches.
+## 1. Automatic schema upgrade
+When using the automatic schema upgrade method, the SQL Server schema migration is performed automatically on the FHIR Server for Azure upgrade.
+The scripts which upgrade the SQL Server schema are part of any upgrade package for the web application and run automatically when the web application launches.
```

Comment:
Minor suggestion: `The scripts which upgrade the SQL Server schema could be part of any upgrade package`